app/build.gradle

```diff
@@ -59,10 +59,10 @@ dependencies {
     implementation "android.arch.navigation:navigation-fragment-ktx:$navigationVersion"
     implementation "android.arch.navigation:navigation-ui-ktx:$navigationVersion"
 
-    implementation 'androidx.recyclerview:recyclerview:1.0.0'
+    implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation 'com.github.abdularis:CircularImageView:v1.2'
     implementation 'com.android.support:cardview-v7:28.0.0'
```

Comment:
Migrate this to androidx.

Reply:
Sure. Thought of doing.

Reply:
@rob729 , Everything migrated to androidx. :fire: